lib/src/screens/dashboard/widgets/sync_indicator.dart

```diff
@@ -27,6 +28,10 @@ class SyncIndicator extends StatelessWidget {
 
         return ClipRRect(
           borderRadius: BorderRadius.all(Radius.circular(15)),
+          child:  GestureDetector(
+            onTap: (){
+              Navigator.of(context, rootNavigator: true).pushNamed(Routes.nodeList);
+            },
```

Comment:
Please take touch callback as widget argument and move out navigation logic from that component to place where that component will be used